Proiect/new-evaluation.css

```diff
@@ -67,5 +80,64 @@
 		}
 
 		li {
-			display: inline;
+			float: left;
+			text-align: left;
+			width: 16%;
+			margin: 2px;
+			min-height: 36px;
+			font-family: Arial;
+			font-size: 15px;
+		}
+
+		#submit {
+			float: right;
+			width: 150px;
+			height: 50px;
+			background-color: #419fe2;
+			color: white;
+			border: none;
+			border-radius: 2px;
+			font-size: 20px;
+			box-shadow: 0px 3px 4px grey;
```

Comment:
`0`, in particular, is the same, regardless of unit of measurement. So, in this case, `0px` == `0%` == `0em` == `0whatever`. In short, `box-shadow: 0 3px 4px grey;` would work just as well, AND you'd be saving TWO WHOLE CHARACTERS worth of disk space to the overall file size! :P